src/eos_gates_windows.js

```diff
@@ -1,28 +1,63 @@
 
 const Lang = imports.lang;
 
+const Flatpak = imports.gi.Flatpak;
 const GLib = imports.gi.GLib;
 const Gio = imports.gi.Gio;
-const EosMetrics = imports.gi.EosMetrics;
+const Gtk = imports.gi.Gtk;
 
 const EosGates = imports.eos_gates;
+EosGates.setupEnvironment();
 
 const WHITELISTED_APPS = [
     // This is an unlikely binary name to run into, so just add it
     // in here for testing purposes...
     { processName: "com.endlessm.test-whitelist.exe" },
 ];
 
+const FLATPAK_APPS = [
+    {
+        regex: /[Ss]potify[Ss]etup.*.exe/,
+        appName: _('Spotify'),
+        flatpakInfo: { remote: 'eos-apps', id: 'com.spotify.Client' }
+    },
+    {
+        regex: /[Ff]irefox\s+[Ss]etup.*.exe/,
+        appName: _('Firefox'),
+        flatpakInfo: { remote: 'eos-apps', id: 'org.mozilla.Firefox' }
+    },
+    {
+        regex: /iTunes.*.exe/,
+        appName: _('Apple iTunes'),
+        flatpakInfo: { remote: 'eos-apps', id: 'com.spotify.Client' },
+        replacementInfo: {
+            appName: _('Spotify'),
+            description: _('a streaming music service')
+        }
+    },
+    {
+        regex: /endless-installer.*.exe/,
+        overrideHelpMessage: _('But do not worry, you are already running Endless OS!')
```

Comment:
@wjt suggested that many people are running this by double-clicking on the .exe file from Endless OS media they have installed - particularly if they have booted a live system - thinking this might be a way to install. In this case of being on a live boot (endless.live_boot in /proc/cmdline) we could perhaps map endless-installer to launch eos-installer.desktop?